content/project/board-election-process.adoc

```diff
@@ -53,61 +57,54 @@ The terms of office for these elected positions are:
 . Use link:https://civs.cs.cornell.edu/[Condorcet Internet Voting Service] as the algorithm to elect multiple people in one vote.
 
-### Nominations
+=== Nominations
 
 The opinion of members of the community at large is highly valued, and the board welcomes additional nominations beyond who we might consider.
 If you feel that a particular person is well suited to help guide Jenkins, please submit a name and the reason for your nomination to jenkinsci-board@googlegroups.com.
 
 The election committee will review all submissions and compile a final list of candidates to be voted on by the community, for the board member seats and officer roles up for election.
-The board reserves the right to omit nominations from the final ballot if they feel that a particular viewpoint is already well represented on the board,
-or if there are other reasons preventing a nominee from being effective in the target role.
+The board reserves the right to omit nominations from the final ballot if they feel that a particular viewpoint is already well represented on the board, or if there are other reasons preventing a nominee from being effective in the target role.
 In the case of rejection, a justification must be provided to the contributor who submitted the nomination.
 The decision may be escalated to the link:/project/governance-meeting/[Jenkins governance meeting] and overridden by voting there.
 
-After receiving and vetting the nominations,
-the election committee is responsible for contacting the nominees and confirming whether they want to run in the election.
+After receiving and vetting the nominations, the election committee is responsible for contacting the nominees and confirming whether they want to run in the election.
 The candidates should also provide their statements, list of affiliations (see <<Corporate Disclosure>>), and account/social media links which may help voters to get more information about them and plans if they are elected to the role.
 This information is published by the election committee before the voting starts.
 
-### Voter sign-up and eligibility
-
-Any Jenkins project and/or community contributor is eligible to vote in the election
-if there is a contribution made before September 01 of the election year.
-_Contribution_ does not mean a _code contribution_,
-all contributions count:
-documentation patches,
-code reviews,
-substantial issue reports,
-issues and mailing list responses,
-social media posts,
-testing,
-etc.
+=== Voter sign-up and eligibility
+
+Any Jenkins project and/or community contributor is eligible to vote in the election if there is a contribution made before September 01 of the election year.
+_Contribution_ does not mean a _code contribution_, all contributions count:
+
+* documentation patches
+* code reviews
+* substantial issue reports
+* issues and mailing list responses
+* social media posts
+* testing
```

Comment:
so this content changed a bit, but i think its fine. though I'd say contributions include, but not limited to type thing (so it doesn't sound like an exhaustive list)

Reply:
okay, i'll just approve it so we can move onto the content one